hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/XceiverClientGrpc.java

```diff
@@ -470,6 +470,11 @@ public XceiverClientReply sendCommandAsync(
     }
   }
 
```

Comment:
A minor comment: Please provide some documentation something like "sendCommandAsync uses this API to decide whether command to work in async mode or in sync mode. In XceiverClientGrpc, read requests are async and write requests are sync. So, this method should return true for write commands and false for read."

So, in overridden EC specific API can document about both write and read should be in sync.

Reply:
Otherwise mostly patch looks straightforward and looks good to me.

Reply:
I have added API doc for both implementation of this method. Thank you for noting this!